src/main/java/edu/kis/vh/nursery/dataStructures/IntLinkedListInterface.java

```diff
@@ -0,0 +1,13 @@
+package edu.kis.vh.nursery.dataStructures;
+
+public interface IntLinkedListInterface {
```

Comment:
The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable. They should have a common interface!